physics/cu_gf_deep.F90

```diff
@@ -1323,9 +1326,9 @@ subroutine cu_gf_deep_run(        &
       do i=its,itf
         if(ierr(i)/=0)cycle
         beta=max(.025,.055-float(csum(i))*.0015)  !.02
-        if(imid.eq.0 .and. xland1(i) == 0)then
-              edtmax(i)=max(0.1,.4-float(csum(i))*.015) !.3)
-        endif
+!        if(imid.eq.0 .and. xland1(i) == 0)then
```

Comment:
In code reviews, we typically suggest that developers remove rather than comment out code because git history saves lines that you might want to re-use in the future and it makes for cleaner code. It's up to you, though, to leave them in if you're experimenting.

Reply:
@grantfirl I will make the code as clean as possible following your reviews. Thanks.